File: plugin/src/main/groovy/com/shogo82148/ribbonizer/FilterBuilder.java
package com.shogo82148.ribbonizer;

import com.android.build.gradle.api.ApplicationVariant;
import com.shogo82148.ribbonizer.resource.Resource;

import java.io.File;
import java.util.function.BiFunction;
import java.util.function.Consumer;

public interface FilterBuilder extends BiFunction<ApplicationVariant, File, Consumer<Resource>> {

}

File: ribbonizer-plugin/src/main/kotlin/com/shogo82148/ribbonizer/FilterBuilder.kt
package com.shogo82148.ribbonizer

import com.android.build.gradle.api.ApplicationVariant
import com.shogo82148.ribbonizer.resource.Resource
import com.shogo82148.ribbonizer.resource.Variant
import org.gradle.api.HasImplicitReceiver
import java.io.File
import java.util.function.BiFunction
import java.util.function.Consumer

interface FilterBuilder: BiFunction<Variant, File, Consumer<Resource>?> {
}